commands/kick.js

```diff
@@ -49,13 +49,14 @@ exports.command = async (message, args, database, bot) => {
   await message.channel.send(responseEmbed);
   const embed = new Discord.MessageEmbed()
   .setColor(0xF62451)
-  .setAuthor(`Case ${insert.insertId} | Kick | ${message.author.username}#${message.author.discriminator}`, member.user.avatarURL())
+  .setAuthor(`Case ${insert.insertId} | Kick | ${member.user.username}#${member.user.discriminator}`, member.user.avatarURL())
```

Comment:
Changed logging in author field to display the kicked person, not the moderator who kicked